addons/sale_loyalty/static/src/views/fields/select_reward_radio/select_reward_radio.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="sale_loyalty.SelectRewardRadioField" owl="1">
```

Comment:
If this template is a copy paste of the RadioField template, we should extend it using t-inherit-mode="primary" (copying it) and adding our own stuff after the label.